hypha/apply/funds/edit_handlers.py

```diff
@@ -63,7 +66,7 @@ def render_as_field(self):
 
         def context(self):
             try:
-                value = getattr(self.instance, self.attr)
+                value = getattr(self.instance, self.panel.attr)
             except AttributeError:
                 self.attr = '__'.join(
```

Comment:
I think we should change it as well to `self.panel.attr` because we are using `self.attr` nowhere.

Reply:
This is not needed, so I've removed it.